src/site/content/en/handbook/markup-code/index.md

```diff
@@ -78,6 +78,16 @@ Show that lines have changed by using the pattern `language/lineNumAdded/lineNum
 </label>
 ```
 
+## Escaping code
+If you need to use double `{` braces in your code examples you will need to
+escape them:
+
+```html
+{% raw %}{% raw %}{% verbatim %}{{ item.num | calculate }}{% endverbatim %}{% endraw %}{% endraw %}
+```
+
+The `{% raw %}{% verbatim %}{% endraw %}` tags will be visible when you stage
```

Comment:
```suggestion
{% Aside %}
The `{% raw %}{% verbatim %}{% endraw %}` shortcodes will be visible when you stage
```
Are we referring to these as tags or shortcodes? If tags, I need to fix elsewhere in the guide.

Reply:
Not sure if it's too much information but...

Lower case things like `{% raw %}` are Nunjucks "blocks".
Uppercase things like `{% Aside %}` are eleventy "shortcodes".

Reply:
Ah-ha. "Tags" it is :)